notebooks/VectorSearch_QuestionRetrieval.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>We're going to be referencing these notebooks in the docs and likely "embedding" them so that users can read through them without having to run them. As a result, I think it's worth checking in the executed notebook w/ the output text filled in.</p><p> </p><p>And since we'll be checking in the notebook with the output cells filled in, I also think we should run an <code>nvidia-smi</code> in the first or second cells so the user can see the GPU architecture the notebook was executed on. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>